pyteal/ast/abi/reference_type_test.py

```diff
@@ -0,0 +1,262 @@
+import pyteal as pt
```

Comment:
Might be a good idea to have a test that calls `compileTeal()`, to enforce that the other steps of the compilation process (like assembly) work as intended.

Reply:
On second thought, I don't think this is needed. I was worried that you weren't testing the full compilation - sometimes bugs crop up towards the end of the process. The integration tests cover this though so feel free to disregard.